api/test_utils.go

```diff
@@ -2,12 +2,17 @@ package api
 
 import (
 	"net/http/httptest"
+	"time"
+
+	"github.com/ajvb/kala/job"
 
 	"github.com/gorilla/mux"
 )
 
 func NewTestServer() *httptest.Server {
 	r := mux.NewRouter()
-	SetupApiRoutes(r)
+	db := &job.MockDB{}
+	cache := job.NewMemoryJobCache(db, time.Hour*1)
```

Comment:
@ajvb ...but when you multiply, do `5*time.Hour`, with the number at the beginning.